lms/templates/ccx/enrollment.html

```diff
@@ -65,7 +65,7 @@
             <tr>
               <td>${member.student}</td>
               <td>${member.student.email}</td>
-              <td><div class="revoke"><i class="icon-remove-sign"></i> Revoke access</div></td>
+              <td><div class="revoke"><i class="fa fa-times-circle"></i> Revoke access</div></td>
```

Comment:
This looks like Font-Awesome, so I changed this, although this CCX stuff is not something I've worked in before...